index.html

```diff
@@ -14,12 +14,13 @@ <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
-      <nav id="desktop-nav">
+      <nav id="desktop-nav" class="nav">
+        <BUTTON type="button" class="nav-btn">X</BUTTON>
         <ul>
           <li><a class="nav-link" href="#projects">Portoflio</a></li>
           <li><a class="nav-link" href="#about-me">About</a></li>
           <li>
-            <a class="nav-link" href="#contact"
+            <a class="nav-link" href="#"
```

Comment:
Kindly add the `href="#contact` for the contact link. One of the project requirements specifies that "When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed." and also that "When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.". Please see the snapshot below

<img width="807" alt="Screen Shot 2021-09-04 at 7 11 19 pm" src="https://user-images.githubusercontent.com/70282171/132102897-21ab2e0d-50df-42ff-a42c-1f651d6e9879.png">